cpu/cc2538/dev/cc2538-rf.c

```diff
@@ -995,10 +1066,12 @@ void
 cc2538_rf_rx_tx_isr(void)
 {
   ENERGEST_ON(ENERGEST_TYPE_IRQ);
+  
+  if(!poll_mode) {
+    process_poll(&cc2538_rf_process);
+  }
 
-  process_poll(&cc2538_rf_process);
-
-  /* We only acknowledge FIFOP so we can safely wipe out the entire SFR */
+  /* We only acknowledge FIFOP or SFD so we can safely wipe out the entire SFR */
```

Comment:
Does this comment actually need updated? In other words, are we still acknowledging the SFD interrupt?